docs/pages/docs/frameworks/nextjs.mdx

```diff
@@ -103,4 +105,147 @@ export default function Page() {
 }
 ```
 
-In addition, here is a complete [project example](https://github.com/tonightpass/kitchen/tree/master/examples/next-typescript) using Kitchen with Next.js.
\ No newline at end of file
+import { FragmentLink, Link } from "@tonightpass/kitchen/next";
+
+
+export const Card = kitchen(Container)`
+  border: 1px solid ${({ theme }) => theme.colors.layout.dark};
+  border-radius: 8px;
+  padding: 16px;
+`;
+
+In addition, here is a complete [project example](https://github.com/tonightpass/kitchen/tree/master/examples/next-typescript) using Kitchen with Next.js.
+
+### Usage
+
+First of all, you need to import the `Link` component from the `@tonightpass/kitchen` package.
+
+```js
+import { Link } from "@tonightpass/kitchen"
+```
+
+### Default
+
+<Playground
+scope={{ Link }}
+  code={`
+  <Link href="/">Return to the home page</Link>
+`}
+/>
+
+### Using onClick
+
+<Playground
+scope={{ Link }}
+  code={`
+  <Link onClick={() => router.push("/")}>Return to the home page</Link>
+`}
+/>
+
+### Disabled
+
+
+<Playground
+scope={{ Link }}
+  code={`
+<Container gap={"small"}>
+  <Link disabled>Disabled link</Link>
+  <Link href={undefined} onClick={undefined}>
+    Disabled link (no href or onClick)
+  </Link>
+  <Link href="#" disabled>
+    {"Disabled link (href=#)"}
+  </Link>
+</Container>
+`}
+/>
+
+### Non text
+
+<Playground
+scope={{ Link, Card }}
+  code={`
+<Link href="/">
+  <Card>This entire div is wrapped in a link</Card>
+</Link>
+`}
+/>
+
+### Variants
+
+<Playground
+scope={{ Link }}
+  code={`
+<Container gap={"small"}>
+  <Container gap={"small"}>
+    <Text size={"small"}>highlight</Text>
+    <Link href="#" variant="highlight">
+      Highlight variant
+    </Link>
+  </Container>
+  <Container gap={"small"}>
+    <Text size={"small"}>primary</Text>
+      <Link href="#" variant="primary">
+        Primary variant
+      </Link>
+  </Container>
+  <Container gap={"small"}>
+    <Text size={"small"}>secondary</Text>
+      <Link href="#" variant="secondary">
+        Secondary variant
```

Comment:
![image](https://user-images.githubusercontent.com/54948363/224284240-77c0b8d2-a3e2-4a46-a41f-e7d2c6907750.png)

Reply:
fix it